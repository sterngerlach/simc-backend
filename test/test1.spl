
{
    int x = 0;
    int y = 0;

    x = 85;
    y = 13;

    iprint(x);
    sprint(" * ");
    iprint(y);
    sprint(" = ");
    iprint(x * y);
    sprint("\n");

    iprint(x);
    sprint(" / ");
    iprint(y);
    sprint(" = ");
    iprint(x / y);
    sprint("\n");
    
    iprint(x);
    sprint(" %% ");
    iprint(y);
    sprint(" = ");
    iprint(x % y);
    sprint("\n");
}

