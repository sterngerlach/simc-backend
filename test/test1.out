85 * 13 = 1105
85 / 13 = 6
85 % 13 = 7
